src/discord-cluster-manager/cogs/leaderboard_cog.py

```diff
@@ -335,11 +339,11 @@ async def leaderboard_create(
         except Exception as e:
             logger.error(f"Error in leaderboard creation: {e}")
             # Handle any other errors
-            await interaction.followup.send(
+            await send_discord_message(
+                interaction,
                 "Error in leaderboard creation.",
                 ephemeral=True,
             )
-            
 
     @discord.app_commands.describe(leaderboard_name="Name of the leaderboard")
```

Comment:
@alexzhang13 I think we should just make it use our `send_discord_message` in all cases, so would replace these `interaction.response.send_message` with it as well

Reply:
Oh there's a weird case with embed=embed or whatever where it doesn't exactly work, but otherwise I can replace the rest.

Reply:
what's the issue with `embed=embed`? Does one of the `followup.send` or `interaction.response.send_message` not support it?

Reply:
Yeah they expect a message, but I can change this as well.

Reply:
We can work around this in a later PR though, for now I think we know what to expect for the embeds (it's always a followup) so we can keep it as is.